### DISCUSSION TITLE
Aula Interagindo com Listas -> Fundamentos de POO

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#133] [Tira Dúvidas] [answered]
## Por que criar a classe CareerItem?
- Usar um vetor de Course na classe Career não seria melhor/+intuitivo doq criar o CareerItem?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @pedro-mileipp , como vai?

Com certeza é uma saída, inclusive a que mostro nos cursos de Modelagem de Domínios e Testes de Unidade, porém, nesta abordagem, mais voltada a acesso à dados, as entidades acabam sendo um reflexo do banco.

Mas não é uma regra e não há certo/errado, você pode mapear sem esta classe associativa se quiser.

**Reply by pedro-mileipp** [reply]
Boa tarde! Obrigado, Balta!
